converters/toZigbee.js

```diff
@@ -4985,6 +4985,8 @@ const converters = {
     wiser_vact_calibrate_valve: {
         key: ['calibrate_valve'],
         convertSet: async (entity, key, value, meta) => {
+            await entity.command('hvacThermostat', 'wiserSmartCalibrateValve',
```

Comment:
Using sendWhenActive with `entity.command` doesn't appear to work as intended, it seems to dispatch it immediately

Reply:
Are you sure you tested on the latest dev branch?

Reply:
I pulled my zigbee-herdsman checkout in my node_modules up to master and built the javascript, but perhaps there are other dependencies i've missed.  I'll update to dev and recheck.

Reply:
> using sendWhenActive with entity.write appears to defer the dispatch, however when the device wakes up it gets stuck in a loop and is constantly sending the write attribute command over the air.

this issue should be fixed now. (https://github.com/Koenkk/zigbee-herdsman/commit/81d8bb9416eb91662c58c70f49ccebdc7fe4efb7)

Change is the in latest-dev, pull the latest dev branch and execute `npm ci` will make sure you are on the correct version.

Reply:
I just double-checked, and I can reproduce this with the latest dev branch, only when sending a cluster command though (attribute read and writes for example work fine).

Reply:
Apparently I should RTFL, the answer was staring me in the face... I had no payload, but options, so i need to insert an empty payload in the call.